components/grid/templates/editor.md

```diff
@@ -22,6 +22,7 @@ You can find the following examples below:
 
 * [Sample editor template that uses a foreign key](#sample-editor-template-that-uses-a-foreign-key)
 
+>note As of version 2.23.0 of Telerik UI for Blazor `EditorTemplate` uses `EditContext` which requires you to also pass `ValueExpression` in case you are <strong>not using</strong> two-way binding (see notes section of [editing/overview]({%slug components/grid/editing/overview%}) article).
```

Comment:
It does not use a context, but rather an edit context is cascading into it because the grid row creates it. It does not always require that you set a value expression, put the condition when you need the expression first, something like "If you anre not using two-waybinding, you need a value epxression".
In fact, if you will be linking the editng notes, you can remove the duplicated content and just state that the row passes the edit context, and you can read more about it in the notes section.

Reply:
Done.